assets/js/modules/analytics-4/utils/report-args.js

```diff
@@ -123,3 +126,31 @@ export function generateReportFilterArgs( filters ) {
 		[ REPORT_ARGS_DATA_FILTERS_KEY ]: JSON.stringify( dataFilters ),
 	};
 }
+
+/**
+ * Returns report args for a sample report.
+ *
+ * @since 1.107.0
```

Comment:
@benbowler could you add a '@since' notation for extracting it to a reusable utility?

Reply:
```suggestion
 * @since 1.107.0
 * @since n.e.x.t Moved from the main analytics-4 datastore file to utils.
```